Description: High dpi scaling does not work... so let's diable it for now...
Author: Martin Wimpress <martin.wimpress@ubuntu.com>

diff --git a/mate-panel/main.c b/mate-panel/main.c
index 8355a22..e3241bc 100644
--- a/mate-panel/main.c
+++ b/mate-panel/main.c
@@ -84,6 +84,9 @@ main (int argc, char **argv)
 
 	gtk_init (&argc, &argv);
 
+    /* FIXME: High dpi scaling does not work... */
+    gdk_x11_display_set_window_scale (gdk_display_get_default (), 1);
+
 	error = NULL;
 	if (!g_option_context_parse (context, &argc, &argv, &error)) {
 		g_printerr ("%s\n", error->message);
